10-habits/progress/internal/tick/tick.go

```diff
@@ -0,0 +1,24 @@
+package tick
```

Comment:
I am not sure if it was a good idea to have a dedicated package.

Reply:
I think this package is slightly wrong. 
We need something, but not this. The `Tick` struct here doesn't really help, imho. 
The `ISOWeek` struct is great. So, I'd keep `ISOWeek()`, I'd rename this package to `isoweek`, and I'd add a single function

```go
package isoweek

// ISO8601 represents a week of a given year.
type ISO8601 struct {
  Week byte // can't exceed 53
  Year int
}

// At returns the ISO8601 week and year at the given timestamp.
// The definition for week 01 is the week with the first Thursday of January in it.
func At(t time.Time) ISO8601 {
  ...
}
```

Reply:
> Week byte // can't exceed 53

The method `t.ISOWeek` from the go package returns an int, I kept the same types. Wdyt?